lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-docker/example-config/configuration.json

```diff
@@ -39,7 +39,7 @@
                 { "usedBy": "NETCONF", "name": "netconf-keystore", "revision": "2017-10-17", "nameSpace": "urn:opendaylight:netconf:keystore"},
                 { "usedBy": "NETCONF", "name": "netconf-node-topology", "revision": "2015-01-14", "nameSpace": "urn:opendaylight:netconf-node-topology"},
                 { "usedBy": "NETCONF", "name":"ietf-netconf", "revision":"2011-06-01", "nameSpace":"urn:ietf:params:xml:ns:netconf:base:1.0"},
-                { "usedBy": "NETCONF", "name": "ietf-netconf-monitoring-extension", "revision": "2013-12-10", "nameSpace": "urn:ietf:params:xml:ns:yang:ietf-netconf-monitoring-extension"},
+                { "usedBy": "NETCONF", "name": "ietf-netconf-nmda", "revision": "2019-01-07", "nameSpace": "urn:ietf:params:xml:ns:yang:ietf-netconf-nmda"},
```

Comment:
ietf-netconf-monitoring-extension was replaced by odl-netconf-monitoring.
 https://git.opendaylight.org/gerrit/c/netconf/+/101857
 
 But maybe it will not be required. Try to find out if we need this one:
 https://git.opendaylight.org/gerrit/c/netconf/+/101856/1